ome_zarr/writer.py

```diff
@@ -147,6 +194,85 @@ def write_multiscales_metadata(
     group.attrs["multiscales"] = multiscales
 
 
+def write_plate_metadata(
+    group: zarr.Group,
+    rows: List[str],
+    columns: List[str],
+    wells: List[str],
+    fmt: Format = CurrentFormat(),
```

Comment:
Why not use ``format`` since name is explicit for other fields?

Reply:
Primarily because `fmt` is the nomenclature used in all the existing APIs in the library. Do you want to capture the rename as an issue?

Reply:
https://github.com/ome/ome-zarr-py/issues/156